### PR TITLE
media-video/dvgrab: fix building with GCC-6

### PR DESCRIPTION
**Pull request by puleglot**
Bug: https://bugs.gentoo.org/594548
Patch: https://github.com/ddennedy/dvgrab/commit/8dd729f2cf4cc5b99ad2e3961419cf71d2dfb843

### CODE REVIEW

File: media-video/dvgrab/dvgrab-3.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	>=media-libs/libdv-0.103
+	>=media-libs/libiec61883-1
+	>=sys-libs/libavc1394-0.5.1
+	jpeg? ( virtual/jpeg:0 )
+	quicktime? ( media-libs/libquicktime )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+	local PATCHES=(
```
Comment (SoapGentoo) [inline media-video/dvgrab/dvgrab-3.5-r1.ebuild]: just make this global please and nix `src_prepare`, make this local gains us nothing

File: media-video/dvgrab/dvgrab-3.5-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+src_configure() {
+	econf \
+		$(use_with quicktime libquicktime) \
+		$(use_with jpeg libjpeg)
+}
+
+src_install () {
+	default
+	dodoc AUTHORS ChangeLog NEWS README TODO
+}
```
Comment (SoapGentoo) [inline media-video/dvgrab/dvgrab-3.5-r1.ebuild]: you should be able to nix `src_install` completely, those docs are part of the default set of `einstalldocs`

Comment (puleglot) [inline media-video/dvgrab/dvgrab-3.5-r1.ebuild]: Yes, indeed. Thanks!

File: media-video/dvgrab/dvgrab-3.5-r1.ebuild (hunk 1 above)
Comment (puleglot) [inline media-video/dvgrab/dvgrab-3.5-r1.ebuild]: Done

**Pull request closed without merging**